src/main/java/org/sunbird/profile/exception/ProfileRequestException.java

```diff
@@ -0,0 +1,22 @@
+package org.sunbird.profile.exception;
+
+/**
+ * @author mahesh.vakkund
+ */
+public class ProfileRequestException extends Exception {
```

Comment:
No need of this new exception class

Reply:
1.Deleted the class for custom exception